embedded/imu_driver/imu_balsaq.cc

```diff
@@ -62,9 +64,13 @@ void ImuBq::loop() {
     ImuMessage msg;
     msg.timestamp = get_current_time();
 
-    const jcc::Vec3 accel_mpss = imu.driver.read_accel_mpss();
-    const jcc::Vec3 angvel_radps = imu.driver.read_gyro_radps();
-    const jcc::Vec3 mag_utesla = imu.driver.read_magnetometer_utesla();
+    jcc::Vec3 accel_mpss = {};
+    jcc::Vec3 angvel_radps = {};
+    jcc::Vec3 mag_utesla = {};
```

Comment:
Good instinct, but leave these uninitialized if you are going to mutate them by reference. This way, we can detect failure to initialize using `Valgrind`. Very annoying, I know.